.github/workflows/ci.yml

```diff
@@ -577,98 +578,102 @@ jobs:
             testing/trino-product-tests/target/*-executable.jar
             client/trino-cli/target/*-executable.jar
           retention-days: 1
+      - id: set-matrix
+        run: |
+          cat <<EOF > .github/test-pt-matrix.yaml
+          config:
+            - default
+            - hdp3
+            - cdh5
+            # TODO: config-apache-hive3
+          suite:
+            - suite-1
+            - suite-2
+            - suite-3
+            # suite-4 does not exist
+            - suite-5
+          jdk:
+            - 11
+          exclude:
+            - config: cdh5
+              ignore exclusion if: >-
```

Comment:
These `ignore exclusion if` don't seem to be used in the Python code.

Reply:
They don't have to be explicitly handled. Because we render the matrix yaml in the `ci.yml` file, the GHA expressions should be evaluated, so there should only be `True/False` values. All matrix values will be combined when expanding in the `expand_matrix` function. There are also tests for that.